builder-images/generate.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+
+generate () {
+  mkdir $IMAGE_NAME
+  cp Dockerfile.template $IMAGE_NAME/Dockerfile
```

Comment:
use `envsubst`:
```
envsubst < Dockerfile.template > $IMAGE_NAME/Dockerfile
```
So then you do not need to use `sed`. Only replace `T_BASE_IMAGE` by `${BASE_IMAGE}` in the template.